lib/discourse_chat/provider/slack/slack_transcript.rb

```diff
@@ -206,9 +206,18 @@ def load_user_data
         json = JSON.parse(response.body)
         return false unless json['ok']
         cursor = json['response_metadata']['next_cursor']
-        @users << json['members']
+        for user in json['members']
+          # Slack uses display_name and falls back to real_name if it is not set
+          if user['profile']['display_name'].empty?
```

Comment:
`blank?` is safer than `empty?` here in case the value is `nil`

Reply:
Shouldn't be possible, but not a problem.  Pushing updates for these comments now.